lib/commands/gesture.js

```diff
@@ -24,21 +24,104 @@ commands.click = async function (el) {
   }
 };
 
+helpers.isSameGestures = function (gestures, candidates) {
```

Comment:
can you please suggest on how to properly import these functions from unit tests in such case?